src/quo2/screens/list_items/dropdown.cljs

```diff
@@ -0,0 +1,71 @@
+(ns quo2.screens.list-items.dropdown
+  (:require [quo.components.text :as text]
+            [quo.previews.preview :as preview]
+            [quo.react-native :as rn]
+            [quo2.components.list-items.dropdown :as quo2]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]))
+
+(def descriptor [{:label   "Icon"
+                  :key     :icon
+                  :type    :select
+                  :options [{:key   :main-icons/placeholder
+                             :value "Placeholder"}
+                            {:key   :main-icons/wallet
+                             :value "Wallet"}]}
+                 {:label "Disabled"
+                  :key   :disabled?
+                  :type  :boolean}
+                 {:label "Default item"
+                  :key   :default-item
+                  :type  :text}
+                 {:label "Border color"
+                  :key   :border-color
+                  :type  :text}
+                 {:label "DD color"
+                  :key   :dd-color
+                  :type  :text}
+                 {:label   "Size"
+                  :key     :size
+                  :type    :select
+                  :options [{:key   :big
```

Comment:
nothing specific to your pr. But do we have an agreed naming/mapping for the size prop.. just to help with consistency across the component library. Seems we have this in most components
If not should we define this? @flexsurfer, @Parveshdhull WDYT? 🤔

Reply:
I would recommend we don't try to give names to the sizes. For few components they are only 3 options :small, :medium, :big. But for some of they, they are way too many, :xxs, :xs, etc.
Also, discussed here: https://github.com/status-im/status-mobile/pull/13830#discussion_r967881050

It will be great, if we can also fix components which are already merged with size names. Maybe we can create issue about that?

Reply:
@Parveshdhull I'd ask for a green light from @flexsurfer or @cammellos first